frontend/src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -2,8 +2,6 @@ import React from "react";
 const PopupWithForm = (props) => {
```

Comment:
**Might be better**

It’s better to **destructure** props not to repeat `props.something` every time

```jsx

function PopupWithForm({isOpen, onClose, name, title, buttonText, children}) {


```